### PR TITLE
Fixes yet another Tequila typo

### LINKED ISSUE #7964: Tequila Sunrise has no icon
See above. it's just empty space.

Great for magic tricks

### PR DESCRIPTION
**Pull request by TheVekter**
Guys, it's spelled "Tequila". One L, not 2. Fucks sake.

Fixes #7964.

### CODE REVIEW

File: code/modules/food&drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -143,6 +143,8 @@
 	icon_state = "vodkabottle"
 	list_reagents = list("vodka" = 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/tequila
+	name = "Caccavo Guaranteed Quality Tequila"
```
Comment (phil235) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: Is this supposed to be here?

Comment (TheVekter) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: Weird, it repeated for whatever reason. I'll remove.

**Comment by TheVekter** [comment]
I know it says no changes, but it's a change to the bottle.dmi

**Comment by phil235** [comment]
merge conflict... somehow

**Comment by TheVekter** [comment]
-Hate-. I'll fix in a bit.

**Comment by TheVekter** [comment]
There's a bunch of stupid unrelated bullshit in here because >Merge conflicts.

**Comment by Razharas** [comment]
I think you fucked up tgstation.dme file, probably pulled when DM was still open so on closure/compile/save it overridden changes with its local memory
Happens to me sometimes, easy to fix

**Comment by TheVekter** [comment]
Holy dick if another fucking tequilla typo comes up I'm going to scream.

**Pull request merged**